component/web/api/src/main/java/org/exoplatform/web/filter/RedefinableResourceFilter.java

```diff
@@ -51,6 +54,15 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             path = path.substring(ctx.length());
         }
 
+        if (StringUtils.contains(path, "favicon")) {
```

Comment:
```suggestion
        if (StringUtils.contains(path, "favicon.ico")) {
```
Can we be more specific ? I'm afraid that checking on favicon string can lead to modify for resources with favicon in name, but not used as favicon

Reply:
Done, thanks.